### ISSUE TITLE
Show warning when file not found

### ISSUE THREAD

**Issue by at15** [arrowrowe/tam#7]
For some dumb users (like me) who often has typo in file names, like write `app` in `ap`
````
"core": {
      "src": ".",
      "dependencies": [
        "angular"
      ],
      "files":[
        "script/ap.js"
      ]
    }
````
tam should show warning when the glob matches nothing
````
[WARN] script/ap.js in package core matches nothing
````
 so the user would know why some file is never loaded and blame on their english teacher instead of tam or themselves. Or have a strict mode other config options to let user enable this kind of warning.

**Comment by arrowrowe** [comment]
Got.

**Comment by arrowrowe** [comment]
Fixed in ed59113bd778b96f5bb7a79d31c667b8d594da1a.

**Comment by at15** [comment]
@arrowrowe www
